### PR TITLE
Fix various misspells

### PR DESCRIPTION
**Pull request by ikoveshnikov**
I've fixed some misspells in changed files in PR #1067. But that PR is too big and it's bad idea to mix everything up, so i've gone through other files, fixed errors there and created this PR.

### CODE REVIEW

File: tempesta_fw/cache.c [hunk 1]
```diff
@@ -345,7 +345,7 @@ tfw_cache_employ_req(TfwHttpReq *req)
 		req->cache_ctl.flags |= TFW_HTTP_CC_CFG_CACHE_BYPASS;
 		return false;
 	}
-	/* cache_fulfill - work as usual in cache mode. */
+	/* cache_fulfil - work as usual in cache mode. */
```
Comment (i-rinat) [inline tempesta_fw/cache.c]: It's spelled `fulfill` everywhere else in the code, including constant names and even configuration parameter: `cache_fulfill`. As far as I understand, both `fulfil` and `fulfill` are accepted. We just need to use either one consistently.

File: tempesta_fw/t/unit/helpers.c [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
  *		Tempesta FW
  *
- * This file contains utils that help to test certain Tempesta FW modules.
+ * This file contains utills that help to test certain Tempesta FW modules.
```
Comment (i-rinat) [inline tempesta_fw/t/unit/helpers.c]: doesn't look right

File: tempesta_fw/cache.c (hunk 1 above)
Comment (ikoveshnikov) [reply tempesta_fw/cache.c]: Yes, American/British English difference. This is the only mistakenly "fixed" spelling, reverted it. Thank you!

File: tempesta_fw/t/unit/helpers.c (hunk 2 above)
Comment (ikoveshnikov) [reply tempesta_fw/t/unit/helpers.c]: Fixed

**Pull request merged**